VellumZero.cs

```diff
@@ -191,33 +257,36 @@ public void Initialize(IHost host)
 
                             // set up shutdown messages
                             StartNotifyTimers();
-                            
+
                             autoRestartTimer.Start();
                         }
-                    };
-                    Host.Bds.OnServerExited += (object sender, EventArgs e) =>
+                    });
+
+                    bds.Process.Exited += (object sender, EventArgs e) =>
                     {
                         // update servers' scoreboards
                         if (_bus != null)
                         {
                             foreach (string server in vzConfig.ServerSync.OtherServers)
                             {
-                                _bus.ExecuteCommand(server, $"scoreboard players reset \"{Host.WorldName}\" \"{vzConfig.ServerSync.ServerListScoreboard}\"");
+                                _bus.ExecuteCommand(server, $"scoreboard players reset \"{_worldName}\" \"{vzConfig.ServerSync.ServerListScoreboard}\"");
                             }
                         }
 
                         // send server disconnect message
                         if (vzConfig.ServerStatusMessages)
                         {
-                            string message = String.Format(vzConfig.VZStrings.ServerDownMsg, Host.WorldName);
-                            if (_discord != null) _discord.SendMessage(message).GetAwaiter().GetResult(); 
+                            string message = String.Format(vzConfig.VZStrings.ServerDownMsg, _worldName);
+                            if (_discord != null) _discord.SendMessage(message).GetAwaiter().GetResult();
                             if (_bus != null) _bus.Broadcast(message);
-                            
+
                         }
 
                         StopTimers();
-                    };                    
-                    Host.Bds.OnShutdownScheduled += (object sender, ShutdownScheduledEventArgs e) =>
+                    };
+
+                    /*
+                    bds.OnShutdownScheduled += (object sender, ShutdownScheduledEventArgs e) =>
```

Comment:
I don't have a practical implementation for subscribing for a scheduled shutdown event in vanilla vellum, another thing I'll have to add before release